webapps/src/main/resources/locale/portlet/Analytics_en.properties

```
@@ -10,6 +10,7 @@ analytics.field.label.statusRank=Tasks: Status Rank
 analytics.field.label.taskId=Tasks: Task ID 
 analytics.field.label.taskLabelIds=Tasks: Task labels 
 analytics.field.label.projectId=Tasks: Project ID 
+analytics.field.label.objectType=Tasks: Object type
```

`objectType` is quite generic, it would be better to make it specific to Tasks if the label is for Tasks only.

```suggestion
analytics.field.label.taskObjectType=Tasks: Object type
```

---

done!